cloudformation/cloudformation.yaml

```diff
@@ -332,6 +332,8 @@ Resources:
           Id: !Sub trigger_state_machine-${Stage}-5
           Input: "{ \"deliveryDateDaysFromNow\": 5 }"
           RoleArn: !GetAtt [ fulfilmentTriggerRole, Arn ]
+        - Id: !Sub salesforceDownloader_${Stage}
+          Arn: !GetAtt SalesforceDownloaderLambda.Arn
```

Comment:
did you test this in CODE? don't you need to set a role to be able to run your lambda from here?